core/src/main/java/com/exadel/aem/toolkit/api/annotations/injectors/InjectorUtils.java

```diff
@@ -0,0 +1,37 @@
+package com.exadel.aem.toolkit.api.annotations.injectors;
+
+import org.apache.sling.api.SlingHttpServletRequest;
+import org.apache.sling.api.resource.Resource;
+import org.apache.sling.api.resource.ValueMap;
+
+/**
+ * Common methods for the injectors
+ */
+public class InjectorUtils {
```

Comment:
To avoid duplicating files across pull-requests, you can open a new pull request "into" the previous pull-request of yours

Reply:
@smiakchilo Should I close this pull request and open in previous one? Or is this advice for future?